tests/integration_test/integration_test.sh

```diff
@@ -125,16 +125,19 @@ PIPELINE_TOOLS_MODE=${4}
 PIPELINE_TOOLS_VERSION=${5}
 TENX_MODE=${6}
 TENX_VERSION=${7}
-SS2_MODE=${8}
-SS2_VERSION=${9}
-TENX_SUBSCRIPTION_ID=${10}
-SS2_SUBSCRIPTION_ID=${11:-"placeholder_ss2_subscription_id"}
-VAULT_TOKEN_PATH=${12}
-SUBMIT_WDL_DIR=${13}
-USE_CAAS=${14}
-USE_HMAC=${15}
-SUBMIT_AND_HOLD=${16}
-COLLECTION_NAME=${17:-"lira-${LIRA_ENVIRONMENT}"}
+OPTIMUS_MODE=${8}
+OPTIMUS_VERSION=${9}
+SS2_MODE=${10}
+SS2_VERSION=${11}
+TENX_SUBSCRIPTION_ID=${12:-"placeholder_10x_subscription_id"}
+OPTIMUS_SUBSCRIPTION_ID=${13:-"placeholder_optimus_subscription_id"}
```

Comment:
This default ID will trigger a [notification](https://github.com/HumanCellAtlas/secondary-analysis/blob/c257c35a0d294786eb7f69e412facc22d9647600/tests/integration_test/optimus_notification_dss_test.json#L3) while the others will not. I think the behavior should be consistent. 

Also since the notification ids do not need to be real and just have to match what is in the test notification json, we could consider renaming them to "optimus_int_subscription_id", "optimus_staging_subscription_id", etc. and then the default could be the integration ones.

Reply:
Fixed